pkg/istio-agent/agent_test.go

```diff
@@ -535,6 +569,30 @@ func copyCerts(t *testing.T, dir string) {
 	}
 }
 
+func copyCertsWithOSRootCA(t *testing.T, dir string) {
+	caRootPath := security.GetOSRootPath()
+	if caRootPath == "" {
+		t.Fatal("OS CA Root Cert could not be found.")
+	}
+	caRootCertSlices := strings.Split(caRootPath, "/")
+	caRootCert := caRootCertSlices[len(caRootCertSlices)-1]
+	if caRootCert == "" {
+		t.Fatal("OS CA Root Cert couldn't be found.")
+	}
+	if err := os.MkdirAll(dir, 0o755); err != nil {
+		t.Fatal(err)
```

Comment:
In cases when it errors out with `err`, you should use `t.Fatalf("why this is bad", err)`